notifications.json

```diff
@@ -0,0 +1,861 @@
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/trait:pagination:page"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:pagination:page_size"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:title"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:to"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:from"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:in"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:unread"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:starred"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:received_before"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:received_after"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:with_snippet"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:with_body"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:thread_id"
+                    },
+                    {
+                        "$ref": "#/parameters/trait:messageSearchQueryString:reference_id"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/message-output"
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/responses/trait:apiErrors:400"
+                    },
+                    "401": {
+                        "$ref": "#/responses/trait:apiErrors:401"
+                    },
+                    "403": {
+                        "$ref": "#/responses/trait:apiErrors:403"
+                    },
+                    "404": {
+                        "$ref": "#/responses/trait:apiErrors:404"
+                    },
+                    "429": {
+                        "$ref": "#/responses/trait:apiErrors:429"
+                    }
+                }
+            }
+        },
+        "/Client/{id}/messages": {
```

Comment:
I would scope admin endpoints under an `/admin` prefix

also, let's leave admin stuff for later and focus on the customers (senders)